nautobot/docs/user-guide/core-data-model/dcim/virtualdevicecontext.md

```diff
@@ -0,0 +1,49 @@
+
+## Overview
+
+![Virtual Device Context Overview](../../../media/models/virtual_device_context_overview.png)
+
+In the above example, the physical device `ams01-edge01` has been "sliced" into two customer VDCs (`cust1-vdc` and `cust2-vdc`). Each of the VDCs contain much of the same data as regular devices, like roles, statuses, tenant relationships, primary IP addresses, etc. As for interfaces, the host device's interfaces can be assigned to one or more VDCs that live on the device. From the example, devices may also retain interfaces that are not assigned to any VDC.
+
+## Key Features
+
+- **Isolation**: Each VDC operates as a separate entity with its own configurations, providing isolation between different contexts.
+
+- **Shared Resources**: VDCs share the physical device's resources, such as processing power and memory, and namely interfaces.
+
+- **Multi-Tenant Support**: Ideal for service providers and enterprises, VDCs help create isolated environments for multiple clients on the same hardware.
+
+- **Flexibility**: VDCs allow for flexible network management, with administrators able to define roles and assign resources based on specific requirements.
+
+## Example Use Cases
+
+The VDC concept goes by several different names, depending on the vendor, and like other Nautobot features, there are nuances to a specific implication, but Nautobot atempts to cover the most broad set of applicable use cases.
+
+Some vendor examples of VDCs include:
+
+1. Cisco Nexus Virtual Device Contexts
+2. Juniper Logical Systems
+3. Arista Multi-instance EOS
+4. Nokia Virtual Service Router
+5. F5 Virtual Clustered Multiprocessing
+
+
+## VDC Fields Overview
+
+| Field         | Type                                              | Description                                          |
+|---------------|---------------------------------------------------|------------------------------------------------------|
+| `name`        | CharField                                         | Name of the VDC.                                     |
+| `device`      | ForeignKey to `dcim.Device`                       | Physical device associated with the VDC.             |
+| `identifier`  | PositiveSmallIntegerField (optional)              | Unique identifier for the VDC from the platform.     |
+| `status`      | StatusField                                       | Operational status of the VDC.                       |
+| `role`        | RoleField (optional)                              | Role assigned to the VDC.                            |
+| `primary_ip4` | OneToOneField to `ipam.IPAddress` (optional)       | Primary IPv4 address of the VDC.                     |
+| `primary_ip6` | OneToOneField to `ipam.IPAddress` (optional)       | Primary IPv6 address of the VDC.                     |
```

Comment:
Hm. We should probably change the implementation here to make these regular ForeignKeys, since a given IPAddress can be assigned to multiple different Devices, each of which may have its own VDCs?

Reply:
Agreed, not sure were 1:1 came from, but I changed it.